src/components/timeline/GlobalTrack.js

```diff
@@ -176,7 +176,10 @@ class GlobalTrackComponent extends PureComponent<Props> {
               onMouseDown: this._onLabelMouseDown,
             }}
           >
-            <h1 className="timelineTrackName">{trackName}</h1>
```

Comment:
Hey @mstange, I traced this `<h1>` back to the very first version of this file, do you remember why you used it? We're considering removing it now because we think it's not a title, but I'd like your feedback. Thanks!

Reply:
I don't mind removing the `h1`. I considered it a label for the track, and `h1` was the closest representation of a "label for a section" that I could think of.

Reply:
Thanks! I think we can remove it now, and then get some feedback from screenreader users to understand how they use the tool.